perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -318,20 +318,32 @@ def launch_setup(context, *args, **kwargs):
 
     # set container to run all required components in the same process
     container = ComposableNodeContainer(
-        name="perception_pipeline_container",
+        name=LaunchConfiguration("container_name"),
         namespace="",
         package="rclcpp_components",
         executable=LaunchConfiguration("container_executable"),
         composable_node_descriptions=[
             crop_box_filter_component,
             ground_filter_component,
         ],
+        condition=UnlessCondition(LaunchConfiguration("use_pointcloud_container")),
         output="screen",
     )
 
+    composable_nodes_loader = LoadComposableNodes(
+        composable_node_descriptions=[
+            crop_box_filter_component,
+            ground_filter_component,
+        ],
+        target_container=LaunchConfiguration("container_name"),
+        condition=IfCondition(LaunchConfiguration("use_pointcloud_container")),
+    )
```

Comment:
Is it okay not to use crop_box and ground_filter if `use_pointcloud_container` is false`?

Reply:
sorry, It's ok.